source/extensions/filters/http/kill_request/BUILD

```diff
@@ -3,10 +3,13 @@ load(
     "envoy_cc_extension",
     "envoy_cc_library",
     "envoy_extension_package",
+    "envoy_package",
 )
 
 licenses(["notice"])  # Apache 2
 
+envoy_package()
```

Comment:
I don't think you need this given the envoy_extension_package declaration.

Reply:
Think this is actually what's also causing docs to fail

Reply:
done.